agent-codeless/src/main/resources/rp-logger-config/diagnostics.appender.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<included>
+    <property scope="system" name="ai.appsvc.logfile.name" value="${ai.logfile.name:-application-insights-extension}" />
+    <property scope="system" name="ai.appsvc.logfile.ext" value="${ai.logfile.ext:-.log}" />
+    <property scope="system" name="ai.appsvc.logfile.old.ext" value="${ai.appsvc.logfile.ext:-.log}.old" />
```

Comment:
I think I'd prefer if these didn't end up in the global System properties.

Would it work to use default (local) scope properties with a bit of duplication between the two xml config files instead?